libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -69,7 +69,7 @@ def _get_search_client(
     semantic_configuration_name: Optional[str] = None,
     fields: Optional[List[SearchField]] = None,
     vector_search: Optional[VectorSearch] = None,
-    semantic_configurations: Optional[SemanticConfiguration] = None,
+    semantic_configurations: Optional[List[SemanticConfiguration]] = None,
```

Comment:
for backwards compatibility we should continue supporting a single SemanticConfiguration as well
```python
semantic_configurations: Optional[Union[SemanticConfiguration, List[SemanticConfiguration]]]
```

Reply:
added